orchestrator/test_runner/Cargo.toml

```diff
@@ -15,20 +15,14 @@ cosmos_gravity = {path = "../cosmos_gravity"}
 gravity_abi = { path = "../gravity_abi" }
 gravity_utils = {path = "../gravity_utils"}
 gravity_proto = {path = "../gravity_proto/"}
-orchestrator = {path = "../orchestrator/"}
```

Comment:
Hadn't we planned to delete the `test_runner` crate entirely in favor of the integration tests? I guess we only did that for the old testnet directory for the module and not the orchestrator?

Reply:
i thought so too, happy to delete if you think we should

Reply:
We can do it later.